ssn/examples/2023ex1.ttl

```diff
@@ -0,0 +1,11 @@
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix sosa: <http://www.w3.org/ns/sosa/> .
+@prefix geo: <http://www.opengis.net/ont/geosparql#> .
+
+<earthAtmosphere_StE> a sosa:Sample ;
+  sosa:isSampleOf <earthAtmosphere> ;
+  geo:hasGeometry [ 
```

Comment:
Memo for later; perhaps this might be sampled at location instead of geometry of location.